### PR TITLE
Better Friend Server Edits 

### PR DESCRIPTION
**Pull request by fncischen**
BFS Edits

### CODE REVIEW

**Review by secantsquared — APPROVED** [review]

**Comment by secantsquared** [comment]
@fncischen Looking great, I will pull down, branch off, and provide my commentary there as to stay out of the way of the workflow and allow you to merge as you and your team leader see fit.  Thanks for sharing, and keeping me in the loop as always.  👍

File: data/users/usersDataModel.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
 module.exports = {
     register,
     login
 }
 
-function register() {
-
+function register(userData) {
+    return db("users").insert({"username": userData.username, "password": userData.password});
```
Comment (kamry-bowman) [inline data/users/usersDataModel.js:10]: this will need to be hashed right?

**Review by kamry-bowman — APPROVED** [review]
Great progress on this! when I try to run locally it has issues parsing but I assume that's a work in progress issue

File: data/users/usersDataModel.js (hunk 1 above)
Comment (fncischen) [reply data/users/usersDataModel.js:10]: Hi @kamry-bowman - thanks for checking in . I actually am hashing it in the user Router!

I am actually having issues parsing it myself and I'd like to see if you could be an extra pair of eyes to figure out where the parsing issues are coming from!

**Comment by fncischen** [comment]
> @fncischen Looking great, I will pull down, branch off, and provide my commentary there as to stay out of the way of the workflow and allow you to merge as you and your team leader see fit. Thanks for sharing, and keeping me in the loop as always. 👍

Thanks @ryanboris appreciate it!

**Pull request merged**